requirements.txt

```diff
@@ -12,4 +12,5 @@ rio-cogeo
 fire
 overpy
 pillow
-geopandas
\ No newline at end of file
+geopandas
+paramiko
```

Comment:
There is also `fiber`, which is build around paramiko. It also handles the connections for you and can even be used to run the same commands on multiple servers. 
There is no need to use that now, just to let you know